File: app/src/main/java/afkt/project/model/bean/ItemStickyBean.java
package afkt.project.model.bean;

import dev.utils.common.DateUtils;

/**
 * detail: Item 实体类
 * @author Ttt
 */
public class ItemStickyBean
        extends AdapterBean {

    // 图片路径
    public String imageUrl;
    // 时间
    public long   time;
    // 时间格式化
    public String timeFormat;
    // 吸附标题
    public String timeTile;

    public ItemStickyBean(
            String title,
            long time
    ) {
        super.title = title;
        this.time = time;

        String format = "yyyy.MM.dd";
        // 进行格式化
        timeFormat = DateUtils.formatTime(time, format);
        // 获取当前时间
        String currentTime = DateUtils.getDateNow(format);
        // 设置标题
        if (currentTime.equals(timeFormat)) {
            timeTile = "今日";
        } else {
            timeTile = DateUtils.formatTime(time, DateUtils.MMdd2);
        }
    }
}

File: app/src/main/java/afkt/project/feature/ui_effect/recy_adapter/item_sticky/ItemStickyBean.kt
package afkt.project.feature.ui_effect.recy_adapter.item_sticky

import afkt.project.model.bean.AdapterBean
import dev.utils.DevFinal
import dev.utils.common.DateUtils

/**
 * detail: Item 实体类
 * @author Ttt
 */
class ItemStickyBean(
    // 标题
    title: String,
    // 时间
    time: Long
) : AdapterBean(title, "") {

    // 时间格式化
    val timeFormat: String

    // 吸附标题
    val timeTile: String

    init {
        val format = DevFinal.TIME.yyyyMMdd_POINT
        // 进行格式化
        timeFormat = DateUtils.formatTime(time, format)
        // 获取当前时间
        val currentTime = DateUtils.getDateNow(format)
        // 设置标题
        timeTile = if (currentTime == timeFormat) {
            "今日"
        } else {
            DateUtils.formatTime(time, DevFinal.TIME.ZH_MMdd)
        }
    }
}